geometry_msgs/Vector3 xyz
geometry_msgs/Vector3 rpy
---
nav_msgs/Odometry odo
